### PR TITLE
Reproducible jdl

### PR DESCRIPTION
**Pull request by mshima**
Forward creationTimestamp option to jhipster-core.
Needed for https://github.com/jhipster/jhipster-core/pull/409

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/generator-base.js [hunk 1]
```diff
@@ -27,6 +27,8 @@ const exec = require('child_process').exec;
 const os = require('os');
 const pluralize = require('pluralize');
 const jhiCore = require('jhipster-core');
+const dateFormatForLiquibase = require('jhipster-core/lib/utils/format_utils').dateFormatForLiquibase;
```
Comment (MathieuAA) [inline generators/generator-base.js]: this isn't to be imported you know :) I believe there's a piece of code somewhere in this repo that does this

Comment (mshima) [reply generators/generator-base.js]: Removed the commit.
I was removing a duplicated code between jhipster-core and generator-jhipster.

Comment (MathieuAA) [reply generators/generator-base.js]: Don't worry, all of this will be refactored very soon when/if JCore and the generator are merged, but thanks!!

**Comment by mshima** [comment]
@MathieuAA what about this PR?

**Pull request merged**